### ISSUE TITLE
Terminal Debugging

### ISSUE THREAD

**Issue by LiuJuiHung** [curdeveryday/vscode-m5stack-mpy#4]
I was wondering can i use this extension to debug my micropython code.
I had already use some connection tools like minicom,ckermit but it's fail.
Can I use this extension to watch the result or error from my micropython code?

**Comment by timmmmmmmmm** [comment]
Agreed. Missing the REPL is a huge disadvantage. When my code isn't working I just get the popup saying 'Run failed' without any feedback.

**Comment by alptbz** [comment]
Yes, I have to agree. It's pretty much useless without.

**Comment by alptbz** [comment]
I implemented a dirty hack, outputting nearly everything to a named output (Output =>M5StackSerial). 
https://github.com/alptbz/vscode-m5stack-mpy/commit/7507e6860fa53ee864be166015cece17602a1025
